### PR TITLE
update txn docs to include race condition note

### PR DESCRIPTION
**Pull request by nickrobin**
addresses https://jira.braze.com/browse/MAPI-52 and discussed in https://brazetechnology.slack.com/archives/G0140E8A210/p1600981291002000

**Description of Change:**
I'm adding a line to detail race condition protections included for transactional API endpoint

### Is this change associated with a Braze feature/product release?
- [X ] Yes (can be released at next release) 
- [ ] No

> If yes, please note the date of the feature release.

---
---

<details>
<summary>PR Checklist</summary>

- [ ] Check that all links work!
- [ ] Ensure you have completed [our CLA](https://www.braze.com/docs/cla/).
- [ ] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is _done and ready to be reviewed for merge_.
- [ ] Tag others as Reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

</details>

<details>
<summary>Writing & Style Quick Tips</summary>

- Use the formatting shown on the [Styling Test Page](https://github.com/Appboy/braze-docs/blob/develop/_docs/_home/styling_test_page.md) to create unique templates and vehicles for your information.
- Use Action verbs!
- Capitalize proper nouns, unique Braze tools and features, custom Braze channels, and all Header titles.
- Redirect when necessary!
- Always speak to the Braze customer - refer to them as “you”, whether they are an Engineer/Developer, a Marketer, or an Admin.
- On Voice: You should aim to be a Friendly Professor. Imagine, if you will, the smartest, but also kindest professor you had. You know, the one who showed you your potential and tested you, but also went slowly in class so you could take great notes and learn some things? Don't be too peppy, but don't be formal. You’re not selling anything - you’re instructing and informing and enabling.

_Consult the [Docs Text Formatting Guide](https://github.com/Appboy/braze-docs/wiki/Special-Formatting) and the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/braze-docs/wiki/Writing-Style-Guide-&-Best-Practices) for more details._

</details>

<details>
<summary>ATTENTION: For PR Reviewers</summary>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then "Docs". A `502` error just means you should refresh until you see the Docs Home page.
</details>

---
---

### CODE REVIEW

File: _docs/_api/endpoints/messaging/send_messages/post_send_transactional_message.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ Authorization: Bearer YOUR_REST_API_KEY
...
 |`trigger_properties`|Optional|Object|Personalization key-value pairs that will apply to the user in this request|
-|`recipient`|Required|Object|The user you are targeting this message to. Please note that given a external user ID that does not already exists in Braze, passing any fields to the `attributes` object will create this user profile in Braze and send to the newly created user.|
+|`recipient`|Required|Object|The user you are targeting this message to. Please note that given a external user ID that does not already exists in Braze, passing any fields to the `attributes` object will create this user profile in Braze and send to the newly created user. Please note, if you send multiple requests to the same user with different data in the `attributes` object, Braze will ensure that standard attributes will be updated synchronously and templated into your message. Custom attributes do not have this same protection, so proceed with caution when updating a user through this API and passing different custom attribute values in quick succession.|
```
Comment (monikkinom) [inline _docs/_api/endpoints/messaging/send_messages/post_send_transactional_message.md:55]: `standard attributes` could include more attributes? Currently we only support first_name, last_name and email.

Comment (nickrobin) [reply _docs/_api/endpoints/messaging/send_messages/post_send_transactional_message.md:55]: they can but thanks for clarify for what we protect against. I'll update

Comment (nickrobin) [reply _docs/_api/endpoints/messaging/send_messages/post_send_transactional_message.md:55]: fwiw i suppose we don't have an exact definition for `standard_attributes` but I think they are commonly thought as of those user attributes you had outlined in ticket (under "User Profile Attributes" here :https://www.braze.com/docs/api/objects_filters/user_attributes_object/)

Comment (nickrobin) [reply _docs/_api/endpoints/messaging/send_messages/post_send_transactional_message.md:55]: semantics aside, i think those 3 fields are the key fields to protect against for now

**Review by KellieHawks — APPROVED** [review]
lgtm!

**Pull request merged**